tests/integration/test_logcollector/test_configuration/test_basic_configuration_location.py

```diff
@@ -23,11 +22,6 @@
 
 wazuh_component = get_service()
 
-if sys.platform == 'win32':
-    prefix = AGENT_DETECTOR_PREFIX
-else:
-    prefix = LOG_COLLECTOR_DETECTOR_PREFIX
-
```

Comment:
About this change... maybe we can introduce this logic in `logcollector.py` and use them in all the callbacks. This way, we won't need to redefine it.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/2318570ff24671281215f030f477b2f9531a31e4